web_src/js/features/citation.ts

```diff
@@ -41,9 +42,9 @@ export async function initCitationFileCopyContent() {
     citationCopyApa.classList.toggle('primary', !isBibtex);
   };
 
-  document.querySelector('#cite-repo-button')?.addEventListener('click', async (e) => {
-    const dropdownBtn = e.target.closest('.ui.dropdown.button');
-    dropdownBtn.classList.add('is-loading');
+  document.querySelector('#cite-repo-button')?.addEventListener('click', async () => {
+    $(modal).modal('show');
```

Comment:
Don't use jquery any more.

Reply:
how to?

Reply:
try to add `display: block` style. in my case it works. if using tailwind then try to add `tw-block`

Reply:
There are plenty of places are using `modal('show')`, I think this can be done in another PR.

Reply:
Depending on https://github.com/go-gitea/gitea/pull/32512/files#diff-95ba7c4096a7a1dee13ee56a3a9ba930cc06aafd7fbc99b5960257bc83c17b0aR76
I think this issue has been fixed.